guarddog/analyzer/sourcecode/code-execution.yml

```diff
@@ -4,105 +4,107 @@ rules:
   - id: code-execution
     languages:
       - python
-    message: 'setup.py file executing code'
+    message: This package is executing OS commands in the setup.py file
+    metadata:
+      description: Identify when an OS command is executed in the setup.py file
     patterns:
       # exec argument must be hardcoded string
       - pattern-either:
-        - patterns:
-          - pattern: exec("...", ...)
-          - pattern: exec($ARG1, ...)
-        - patterns:
-          - pattern: exec("...". ...)
-          - pattern: exec($ARG1. ..., ...)
-        - patterns:
-          - pattern: exec("..." + ...)
-          - pattern: exec($ARG1 + ..., ...)
+          - patterns:
```

Comment:
same, do we have automated tool to enforce this one?

Reply:
same response, reformatted through Intellij but we don't have anythingI thnk